developers/weaviate/modules/reader-generator-modules/generative-anyscale.md

```diff
@@ -216,30 +216,30 @@ You can add both `title` and `summary` to the prompt by enclosing them in curly
 #### Example - single result
 
 Here is an example of a query where:
-* we run a vector search (with `nearText`) to find articles about "Italian food"
-* then we ask the generator module to describe each result as a Facebook ad.
-  * the query asks for the `summary` field, which it then includes in the `prompt` argument of the `generate` operator.
+* we get a podcast clip (with limit 1)
```

Comment:
Consider parallel list structure: 

* We get a podcast clip (with limit 1).
* We ask the generator module to summarize the content into one sentence.
* The query

Reply:
Closing for now for the same reason as below.